ledger/acctupdates.go

```diff
@@ -2034,6 +2044,10 @@ func (au *accountUpdates) commitSyncer(deferedCommits chan deferredCommit) {
 
 // commitRound write to the database a "chunk" of rounds, and update the dbRound accordingly.
 func (au *accountUpdates) commitRound(offset uint64, dbRound basics.Round, lookback basics.Round) {
+	var stats telemetryspec.AccountsUpdateMetrics
+
+	updateStats := au.logAccountUpdateMetrics && time.Now().Sub(au.lastMetricsLogTime) >= au.logAccountUpdateFreq
```

Comment:
The duration of this method is known.. so you don't want to skew the metrics timing based on that.
i.e. the update time you place in `au.lastMetricsLogTime` should be the same as in this query:
```golang
var updateStats bool
if au.logAccountUpdateMetrics {
    now := time.Now()
    if now.Sub(au.lastMetricsLogTime) >= au.logAccountUpdateFreq {
       updateStats = true
       au.lastMetricsLogTime = now
    }
}
```